core/dbt/events/adapter_endpoint.py

```diff
@@ -56,3 +57,10 @@ def critical(self, msg, *args):
             name=self.name, base_msg=str(msg), args=list(args), node_info=get_node_info()
         )
         fire_event(event)
+
+    @staticmethod
+    def set_adapter_dependency_log_level(package_name, level):
```

Comment:
I know you implemented this in `dbt-snowflake`. Do we need to do the same for `dbt-postgres`? Or is there already a default where we don't need to make any changes?

Reply:
I don't believe there's been a need for it in dbt-postgres as of yet